src/internal/storage/kv/obj_adapter2.go

```diff
@@ -0,0 +1,97 @@
+package kv
+
+import (
+	"context"
+	"io"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/miscutil"
+	"github.com/pachyderm/pachyderm/v2/src/internal/stream"
+
+	"gocloud.dev/blob"
+	"gocloud.dev/gcerrors"
+)
+
+type ObjectStore2 struct {
```

Comment:
The type in `obj_adapter.go` is `objAdapter`, so you could potentially just call this `ObjectStore` and drop the `2`

Reply:
I was thinking we would rename this to something that makes more sense after we delete the other one.